modules/util/path.go

```diff
@@ -14,3 +17,18 @@ func EnsureAbsolutePath(path string, absoluteBase string) string {
 	}
 	return filepath.Join(absoluteBase, path)
 }
+
+// GetDirectorySize returns the dumb disk consumption for a given path
+func GetDirectorySize(path string) (int64, error) {
+	var size int64
+	err := filepath.Walk(path, func(_ string, info os.FileInfo, err error) error {
+		if err != nil {
```

Comment:
this is not needed anymore, `err` will be returned below and in case of error `info` will be `nil`